src/simplewallet/simplewallet.cpp

```diff
@@ -5353,6 +5731,7 @@ bool simple_wallet::set_daemon(const std::vector<std::string>& args)
     else
     {
       m_wallet->set_trusted_daemon(false);
+#if !defined(SEKRETA)  // Pointing to local doesn't imply trusted
```

Comment:
is_local_address now detects .onion and .i2p suffixes. If that is not enough, is there a set of suffixes or known formats that would be enough, or can the address be just "127.0.0.1:12345" and yet be a bridge to the outside ?